docs/custom-flows/email-password.mdx

```diff
@@ -34,105 +34,104 @@ A successful sign-up consists of the following steps:
+    setLoading(true)
+    if (!isLoaded) return
 
     try {
       const completeSignUp = await signUp.attemptEmailAddressVerification({
-        code,
-      });
-      if (completeSignUp.status !== "complete") {
-        /*  investigate the response, to see if there was an error
-         or if the user needs to complete more steps.*/
-        console.log(JSON.stringify(completeSignUp, null, 2));
-      }
+        code
+      })
+
       if (completeSignUp.status === "complete") {
         await setActive({ session: completeSignUp.createdSessionId })
-        router.push("/");
       }
-    } catch (err: any) {
-      console.error(JSON.stringify(err, null, 2));
+      setLoading(false)
+      router.push("/")
     }
-  };
+    catch (err: any) {
+      setErrors(err.message)
+      setLoading(false)
+    }
+  }
+
+  if (verifying) {
+    return (
+      <form onSubmit={handleVerify}>
+        <label id="code">Code</label>
+        <input value={code} id="code" name="code" onChange={(e) => setCode(e.target.value)} />
+        <button type="submit">Complete Sign Up</button>
+      </form>
+    )
+  }
 
   return (
-    <div>
-      {!pendingVerification && (
-        <form>
-          <div>
-            <label htmlFor="email">Email</label>
-            <input onChange={(e) => setEmailAddress(e.target.value)} id="email" name="email" type="email" />
-          </div>
-          <div>
-            <label htmlFor="password">Password</label>
-            <input onChange={(e) => setPassword(e.target.value)} id="password" name="password" type="password" />
-          </div>
-          <button onClick={handleSubmit}>Sign up</button>
-        </form>
-      )}
-      {pendingVerification && (
-        <div>
-          <form>
-            <input
-              value={code}
-              placeholder="Code..."
-              onChange={(e) => setCode(e.target.value)}
-            />
-            <button onClick={onPressVerify}>
-              Verify Email
-            </button>
-          </form>
-        </div>
-      )}
-    </div>
-  );
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label htmlFor="email">Email address</label>
+        <input id="email" type='email' name="email" value={emailAddress} onChange={(e) => setEmailAddress(e.target.value)} />
+      </div>
+      <div>
+        <label className="block text-sm mt-8" htmlFor="password">Password</label>
+        <input id="password" type='password' name="password" value={password} onChange={(e) => setPassword(e.target.value)} />
+      </div>
+      <div>
+        <button type="submit">Verify Email</button>
+      </div>
+    </form>
+
+  )
 }
-```
-```tsx filename="app/sign-up/[[...sign-up]].page.tsx"
-"use client"
+````
+```tsx filename="pages/sign-up/[[...index]].tsx"
+
 import { useState } from "react";
 import { useSignUp } from "@clerk/nextjs";
-import { useRouter } from "next/navigation";
+import { useRouter } from "next/router";
 
 export default function SignUpForm() {
```

Comment:
I would make sure the comments for App Router example match the comments for Pages Router example.